tests/data/SPECS/buildrequires.spec

```diff
@@ -7,6 +7,7 @@ License: GPL
 Distribution: RPM test suite.
 Source0: buildrequires-1.0.tar.gz
```

Comment:
why don't we drop this tarball? or modify it to have just one file

Reply:
I almost did (along with bunch of other cleanups) but as the buildrequires.txt is inside it... seemed more trouble than worth right now.